### ISSUE TITLE
Tinder like interface for collectors to buy NFTs

### ISSUE THREAD

**Issue by yangwao** [kodadot/nft-gallery#2099]
Would be nice to start having an easy interface to collect NFTs on KodaDot.
Feel free to choose framework if you find more up to date one

### Spec
- add new route `/swipe`
- add with each swipe randomness function to pick random NFT which is for SALE
- add buttons to execute buy
- add keyboard shortcuts to swipe left and right with arrows
- left is to next card (swipe left)
- right will trigger emote (swipe right)
- spacebar will make buy (or swipe to the top)
- swipe bottom, down arrow open artist profile
- make it able to create a view on top of the collection, like suffix to the end of collection name `/swipe` 

### Ref
- https://news.shardlabs.io/creating-swipeable-cards-in-vue-js-49e4ea8ed8a3
- https://www.npmjs.com/package/vue-swipy
- https://shard-labs.github.io/vue-swipy/

**Comment by petersopko** [comment]
🙋🏼‍♂️ this seems like both fun feature + something where I could get my hands dirty with proper Vue. If the p5 label stays, and it's okay for me to start on Tuesday with this, I'd love to do it.

**Comment by kkukelka** [comment]
I'll give it a shot. Will take a while though.

**Comment by XyloDrone** [comment]
would this only work on mobile or? 
also considering where to add this... in top bar of explore?

**Comment by yangwao** [comment]
> would this only work on mobile or? also considering where to add this... in top bar of explore?

yeah, swiping doesn't work well outside mobile tbh, so yeah mobile only probably.. and still thinking if it should be pushed to mobile app, rather mobile version of web kodadot
